addons/editor-cleanup-plus/userscript.js

```diff
@@ -0,0 +1,142 @@
+import UndoGroup from "../editor-devtools/UndoGroup.js";
+import { getVariableUsesById, getOrderedTopBlockColumns } from "../editor-devtools/utils.js";
+
+export default async function ({ addon, console, msg, safeMsg: m }) {
+  const blockly = await addon.tab.traps.getBlockly();
+  const getWorkspace = () => blockly.getMainWorkspace();
```

Comment:
```suggestion
  const getWorkspace = () => addon.tab.traps.getWorkspace();
```
devtools was canged to use this API in #7892.